controllers/operator_controller.go

```diff
@@ -175,6 +175,17 @@ func (r *OperatorReconciler) reconcile(ctx context.Context, op *operatorsv1alpha
 		return ctrl.Result{}, err
 	}
 
+	if isBundleDepStale(existingTypedBundleDeployment) {
+		apimeta.SetStatusCondition(&op.Status.Conditions, metav1.Condition{
+			Type:               operatorsv1alpha1.TypeReady,
+			Status:             metav1.ConditionUnknown,
+			Reason:             operatorsv1alpha1.ReasonInstallationStatusUnknown,
+			Message:            fmt.Sprintf("waiting for bundleDeployment %q status to be updated. BundleDeployment conditions out of date.", existingTypedBundleDeployment.Name),
+			ObservedGeneration: op.GetGeneration(),
+		})
+		return ctrl.Result{}, nil
+	}
+
```

Comment:
Should we put this inside the `mapBDStatusToReadyCondition` function?

Reply:
Fine either ways, just made it separate to make it clear that we are not mapping the BD status since its stale.